### ISSUE TITLE
Unable to complete bounties.

### ISSUE THREAD

**Issue by LilyArlatto** [ejektaflex/Bountiful#43]
I have never used this mod on its own but it seems like it has very few bugs and should be working properly, so I thought I should make a new issue. Whenever this mod is in the "Life in the village" modpack on the Twitch desktop app, there must be some weird conflict because it literally just won't let me complete the bounty at all. I tried spamming right click, I tried dropping everything, and I can't put the items in the bounty board so that's obviously not how you do it. I really wish it would just work because my friend made a server using this modpack and it seems the bounties are almost an integral part to the "questing" system in the modpack.

**Comment by ejektaflex** [comment]
Hey there! I'll try and download the modpack to see if I can reproduce this. Is there anything that you were wearing / did you have the bounty in your right hand / was your left hand free? There is also a config option that you can change in the modpack folder, inside `config/bountiful/bountiful.cfg`, titled `Cash In At Bounty Board?`. You can reverse that setting and see if the alternative way of turning in bounties works.

Otherwise, it would be great if I could also have a log showing what the game was sending to the output. That can be found in the modpack folder, inside `logs/latest.log`. You can add that file here by just dragging it into the comment box.

Thank you!

**Comment by ejektaflex** [comment]
For reference, it seems like the log that somebody else posted contained this:

```
[12:38:12] [Server thread/ERROR] [FML]: Exception caught during firing event net.minecraftforge.event.entity.player.PlayerInteractEvent$RightClickBlock@1fd95384:
java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
	at ejektaflex.bountiful.item.ItemBounty.getItemStackDisplayName(ItemBounty.kt:89) ~[ItemBounty.class:?]
	at net.minecraft.item.ItemStack.getDisplayName(ItemStack.java:617) ~[aip.class:?]
	at furgl.improvedHoes.common.ImprovedHoes.isRegisteredHoe(ImprovedHoes.java:35) ~[ImprovedHoes.class:?]
	at furgl.improvedHoes.common.event.PlayerInteractEvents.onEvent(PlayerInteractEvents.java:19) ~[PlayerInteractEvents.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_917_PlayerInteractEvents_onEvent_PlayerInteractEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) [EventBus.class:?]
	at net.minecraftforge.common.ForgeHooks.onRightClickBlock(ForgeHooks.java:1124) [ForgeHooks.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.processRightClickBlock(PlayerInteractionManager.java:433) [or.class:?]
	at net.minecraft.network.NetHandlerPlayServer.processTryUseItemOnBlock(NetHandlerPlayServer.java:741) [pa.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:55) [ma.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:11) [ma.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_211]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_211]
	at net.minecraft.util.Util.runTask(SourceFile:529) [h.class:?]
	at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397) [nz.class:?]
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_211]
```

It will be interesting to see if the same message pops up.

**Comment by wierzejs** [comment]
I am playing the same pack. Didn't yet have time to play around in the config but I got console output of what happens when I right click it. No items in my off hand

https://pastebin.com/YRaKrbii

**Comment by ejektaflex** [comment]
Yep, it's a localization bug. Will be fixed in the next update, when I get around to it.

**Comment by lolsteamroller** [comment]
Similar error for me, couldn't start server, had to remove the mod sadly. Waiting for fix.

**Comment by ejektaflex** [comment]
@lolsteamroller This is a localization bug, so you wouldn't get an error disallowing you from starting the server. Delete your Bountiful config and try again. This error only occurs in specific instances where another mod tries to get the localized name of a Bounty item. In your case, you probably have a bad JSON config.

**Comment by ejektaflex** [comment]
@LilyArlatto and everyone else: This should be fixed in f0b4eef. The fix will be rolled out in v2.2.1 shortly. Thank you!
